src/routing/Routing.tsx

```diff
@@ -22,35 +24,38 @@ const AboutUsPage = lazy(() => import('src/pages/AboutUsPage'));
 const Routing = () => {
   return createBrowserRouter(
     createRoutesFromElements(
-      <Route path={ROUTES.HOME} element={<Layout />}>
-        <Route path={ROUTES.HOME} element={<HomePage />} />
-        <Route path={ROUTES.ABOUT_US} element={<AboutUsPage />} />
-        <Route path={ROUTES.ROADMAPS} element={<RoadmapsPage />} />
-        <Route path={`${ROUTES.ROADMAPS}/:id`} element={<RoadmapPage />} />
-        <Route
-          path={ROUTES.REGISTER}
-          element={
-            <PublicRoute restricted path={ROUTES.PROFILE}>
-              <RegistrationPage />
-            </PublicRoute>
-          }
-        />
-        <Route
-          path={ROUTES.LOGIN}
-          element={
-            <PublicRoute restricted path={ROUTES.PROFILE}>
-              <LoginPage />
-            </PublicRoute>
-          }
-        />
-        <Route
-          path={ROUTES.PROFILE}
-          element={
-            <PrivateRoute>
-              <EditProfilePage />
-            </PrivateRoute>
-          }
-        />
+      <Route>
+        <Route path={ROUTES.HOME} element={<Layout />}>
+          <Route path={ROUTES.HOME} element={<HomePage />} />
+          <Route path={ROUTES.ABOUT_US} element={<AboutUsPage />} />
+          <Route path={ROUTES.ROADMAPS} element={<RoadmapsPage />} />
+          <Route path={`${ROUTES.ROADMAPS}/:id`} element={<RoadmapPage />} />
+          <Route
+            path={ROUTES.REGISTER}
+            element={
+              <PublicRoute restricted path={ROUTES.PROFILE}>
+                <RegistrationPage />
+              </PublicRoute>
+            }
+          />
+          <Route
+            path={ROUTES.LOGIN}
+            element={
+              <PublicRoute restricted path={ROUTES.PROFILE}>
+                <LoginPage />
+              </PublicRoute>
+            }
+          />
+          <Route
+            path={ROUTES.PROFILE}
+            element={
+              <PrivateRoute>
+                <EditProfilePage />
+              </PrivateRoute>
+            }
+          />
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
```

Comment:
why don't you just bring not-found route in layout one? I think you don't need to add extra route on 27 line